.buildkite/commands/run-ui-tests.sh

```diff
@@ -35,7 +35,12 @@ install_cocoapods
 echo "--- 🧪 Testing"
 xcrun simctl list >> /dev/null
 rake mocks &
+set +e
 bundle exec fastlane test_without_building name:"$TEST_NAME" device:"$DEVICE" ios_version:"$IOS_VERSION"
+TESTS_EXIT_STATUS=$?
+set -e
 
 echo "--- 📦 Zipping test results"
 cd fastlane/test_output/ && zip -rq WooCommerce.xcresult.zip WooCommerce.xcresult && cd -
+
+exit $TESTS_EXIT_STATUS
```

Comment:
What do you think of also adding the `echo "^^^ +++"` on non-zero exit code (like is done [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/4416fb9cc758fc6d9f650b16d535b7c34511b69b/.buildkite/commands/run-unit-tests.sh#L24-L28)) so that Buildkite will have the "Testing" section expanded and open by default — instead of the "Zipping test results" one — when the test failed, to avoid confusion for anyone checking the logs and seeing the failure looking like it was on the zipping instead of the testing step?

See [Buildkite docs here](https://buildkite.com/docs/pipelines/managing-log-output#collapsing-output:~:text=If%20you%27d%20like%20to%20open%20the%20previously%20defined%20group%2C%20use%20%5E%5E%5E%20%2B%2B%2B.%20This%20is%20useful%20if%20a%20command%20within%20a%20group%20fails%2C%20and%20you%27d%20like%20to%20have%20the%20group%20already%20open%20when%20you%20view%20the%20log.)

Reply:
Great idea. I've copied this feature over in https://github.com/woocommerce/woocommerce-ios/pull/7647/commits/0dcfec050ed42aa860fbf5ed6af69d94654549db